lib/data/replacements.json

```
@@ -211,6 +212,18 @@
       }
     ]
   },
+  "eslint-plugin-eslint-comments-to-maintained-fork": {
+    "description": "Replace `eslint-plugin-eslint-comments` with a maintained fork: `@eslint-community/eslint-plugin-eslint-comments`.",
+    "packageRules": [
+      {
+        "matchDatasources": ["npm"],
+        "matchPackageNames": ["eslint-plugin-eslint-comments"],
+        "matchCurrentVersion": "^4.0.0",
+        "replacementName": "@eslint-community/eslint-plugin-eslint-comments",
+        "replacementVersion": "4.4.0"
```

```suggestion
        "replacementVersion": "3.2.0"
```

This exists per https://www.npmjs.com/package/@eslint-community/eslint-plugin-eslint-comments/v/3.2.0